content/docs/practices/naming.md

```diff
@@ -44,7 +44,7 @@ Use labels to differentiate the characteristics of the thing that is being measu
 Do not put the label names in the metric name, as this introduces redundancy
 and will cause confusion if the respective labels are aggregated away.
 
-CAUTION: <b>CAUTION:</b> Remember that every unique key-value label pair
+**CAUTION:** Remember that every unique key-value label pair
```

Comment:
The initial `CAUTION` here triggers a special "caution" markup block and doesn't actually show up in the text. See http://prometheus.io/docs/practices/naming/. It'll need to be preserved.